### DISCUSSION TITLE
Table selection is not available

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#2067] [Help] [answered]
when I use the Table or repeater function I can not select a data source. what do I need to do for the tables/views to populate?
![image](https://user-images.githubusercontent.com/80058238/126677277-91156846-0947-4f34-9ec5-58c5d5a63a92.png)

**Comment by GuilhermeHirsch** [comment, marked as answer]
Hello, @AndrewJRamay! Have you inserted the "Data Provider" component?

**Reply by KernOil** [reply]
Hi @GuilhermeHirsch, yes I have a data provider component now, however it is only showing internal data. I have a MySQL connection I would like to view with it and some views. but nothing shows up.

**Reply by KernOil** [reply]
I think I have it now I must use the sort column selection to return any data, Thank you for your help!

**Reply by KernOil** [reply]
@GuilhermeHirsch How can I edit a row in the table?

**Reply by GuilhermeHirsch** [reply]
You can create automatic screens. These screens are based on your tables, and have the functions of "view", "update and delete" and "new record creation".

![Sem título](https://user-images.githubusercontent.com/33108596/126680501-be57863e-96b2-41b1-a449-b0f791b2e213.jpg)

**Reply by GuilhermeHirsch** [reply]
If you need more details on the subject, I can go deeper.

**Reply by KernOil** [reply]
ok i understand now Thank you

**Reply by GuilhermeHirsch** [reply]
Excellent, @AndrewJRamay! I am here to help! Have a great day!
